.github/workflows/testDocsUpdate.yml

```diff
@@ -0,0 +1,103 @@
+name: Test Docs Update
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        description: "Prerelease"
+        required: true
+        default: false
+        type: boolean
+      draft:
+        description: "Draft"
+        required: true
+        default: false
+        type: boolean
+      version-increment-type:
+        description: 'Which part of the version to increment:'
+        required: true
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+        default: 'patch'
+
+
+jobs:
+  release:
+    name: Test Docs Update
+    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            node-version: [18.x]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AUTOMATION_USER_TOKEN }}
+          fetch-depth: 0
+      
+      # - uses: DevCycleHQ/release-action/prepare-release@main
```

Comment:
delete all this?

Reply:
i'm going to delete the whole file as soon as i get it working